### DISCUSSION TITLE
Pagination examples

### DISCUSSION THREAD

**Discussion by drbswa** [POPWorldMedia/POPForums#221] [General]
Are there any examples of how to use pagination within a forum?

**Comment by jeffputz** [comment]
There are several layers to this, which part do you want to discuss?

**Reply by drbswa** [reply]
It seems as though there is some kind of PagedTopicContainer function that exists. I'm rather new to implementing a lot of these things and was just hoping there might be an example somewhere of how it's structured/implemented.

**Reply by jeffputz** [reply]
Sure... load up the solution and start in the controller method that fetches the model, and work your way back by right-clicking and "go to implementation" and you'll follow it all the way up to the actual SQL calls.
